ansible/Makefile

```diff
@@ -9,7 +9,7 @@ download-kubespray:
 	poetry run ansible-playbook -i localhost, -c local download_kubespray.yml
 
 .PHONY: download-cli-binaries
-download-cli-binaries:
+download-cli-binaries: download-ansible-roles
```

Comment:
if you add that dependency, I suppose you can remove the `download-ansible-roles` dependency from the `download` task above, to avoid having it download things twice.